site-src/api-types/inferencepool.md

```diff
@@ -7,12 +7,26 @@
 
 ## Background
 
-InferencePool is
+The InferencePool at its core is a logical grouping of compute, expressed in the form of Pods (typically model servers), akin to a K8s Service. The InferencePool would deploy its own routing, and offer administrative configuration to the Platform Admin. 
```

Comment:
Consider replacing "The InferencePool at its core is a logical grouping of compute, expressed in the form of Pods (typically model servers), akin to a K8s Service." with "The InferencePool resource is a logical grouping of compute resources, e.g. Pods, that run model servers."